web/src/styles/base/global.scss

```diff
@@ -87,7 +87,7 @@ ol {
 
 .usa-link:visited,
 .usa-prose > a:visited {
-  color: $primary-darker !important;
+  color: $accent-cool-darkest !important;
```

Comment:
[pebble] In storybook, the colors are rendered with the lightest colors on the left, and the shades get darker as you move to the right.

I'd assume `primary` would get swapped with `accent-cool-darker`, `primary-dark` with `accent-cool-darkest` and `primary-darker` with `accent-cool-more-dark`

<img width="479" alt="Screenshot 2023-09-01 at 10 13 10 AM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/72581979/c63a46fe-e40f-43f1-9d63-d319c652da94">
<img width="426" alt="Screenshot 2023-09-01 at 10 13 23 AM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/72581979/2699417e-dfbb-4d5c-8241-db7ca7101458">

http://nj-bfs-design-storybook.s3-website-us-east-1.amazonaws.com/?path=/story/atoms-colors--colors

Reply:
The AC of this ticket specifically says to use `accent-cool-darkest` and `accent-cool-more-dark` on hover. Happy to message design about this, but that's not how I read this.